content/10.diagnostics.md

```diff
@@ -10,6 +10,8 @@ A range of diagnostic approaches from a methodological standpoint are being or c
 ### Molecular Tests
 
 Molecular tests are often carried out from nasopharyngeal cavity samples, including throat wash and saliva [@doi:10/ggp4qx].
+They have also been shown to work in one study using stool samples [@doi:10.1002/jmv.25742].
+This has been applied to broader community surveillance rather than for patient diagnostics, in studies that tested wastewater. [@doi:10.1016/j.scitotenv.2020.138764]
```

Comment:
```suggestion
This has been applied to broader community surveillance, rather than for patient diagnostics, in studies that tested wastewater [@doi:10.1016/j.scitotenv.2020.138764].
```